test/packages/parallel/apache/_dev/test/config.yml

```diff
@@ -0,0 +1,2 @@
+system:
+  parallel: true
```

Comment:
Could we also enable parallel tests in the sql input package? I tried locally and seems to work well.

Reply:
Sure, I'll add this same configuration file to sql_input :+1: